ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -20,6 +20,7 @@ import getAgent from '../../../../utilities/undiciAgent';
 
 import LivePageLayout from './LivePageLayout';
 import extractHeaders from '../../../../../src/server/utilities/extractHeaders';
+import isValidPageNumber from '../../../../utilities/pageQueryValidator';
```

Comment:
Is this logic necessary if the request is filtered out downstream via Belfrage?

Reply:
Thanks for the feedback, Harvey.
I'd venture that it is. The rationale behind this being that validation is performed
independent of the validation other services are performing.

Reply:
Discussed in call, LGTM